app/src/components/market/profile/view.tsx

```diff
@@ -99,17 +96,20 @@ const ViewWrapper = (props: Props) => {
         <Table head={renderTableHeader()}>{renderTableData()}</Table>
         <ButtonContainerStyled>
           {userHasShares && (
-            <Button backgroundColor={theme.colors.secondary} onClick={() => props.handleSell()}>
+            <Button
```

Comment:
Super low priority: using an anchor link is better ux (you can middle click, hover to see the link, copy link address, etc.)

Reply:
I added an anchor button , I think it looks pretty good ...